core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2264,8 +2264,11 @@ private void loadMetadataInternal(RpcContext rpcContext, LockedInodePath inodePa
             listOptions.setRecursive(false);
           }
           UfsStatus[] children = ufs.listStatus(ufsUri.toString(), listOptions);
+          // children can be null if the pathname does not denote a directory
```

Comment:
Unless the user is directly calling `LoadMetadata`, which is a command that we have deprecated.  All the other calls to `loadMetadataInternal` has catch exception and logging around it.  Is that graceful enough?

Reply:
Could the call below (https://github.com/Alluxio/alluxio/pull/7852/files#diff-7f82122f923de058e3c4a20b43cd7e4eR2293) cause us to skip loading metadata for all children if listing one child fails? This could give strange results if a user calls `ls` on a big directory and only sees the sub-paths that were scanned before the one broken directory.